### ISSUE TITLE
Constraint Issues in Windows 7

### ISSUE THREAD

**Issue by Digital-Thor** [vsivsi/meteor-file-collection#56]
Thanks for making the "Fiat" version of a File Collection package as you said in your readme!  I'm looking forward to using it.  The [sample app](https://github.com/vsivsi/meteor-file-sample-app) worked well on my Meteor Windows 7 environment until I tried to incorporate and test it within my existing app.  
The errors that I got when I tried to add `meteor-file-collection` are:

```
meteor add vsivsi:file-collection@1.1.1
 => Errors while adding packages:

While selecting package versions:
error: Conflict: Constraint coffeescript@1.0.6 is not satisfied by coffeescript 1.0.6-rc.0.
Constraints on package "coffeescript":
* coffeescript@=1.0.6-rc.0 <- top level
* coffeescript@1.0.6 <- vsivsi:file-collection 1.1.1

Conflict: Constraint webapp@1.2.0 is not satisfied by webapp 1.2.0-rc.1.
Constraints on package "webapp":
* webapp@=1.2.0-rc.1 <- top level
* webapp@1.2.0-rc.0 <- meteor-platform 1.2.2-rc.0
* webapp@1.2.0-rc.1 <- autoupdate 1.2.0-win.0 <- meteor-platform 1.2.2-rc.0
* webapp@1.2.0-rc.0 <- routepolicy 1.0.5-rc.0 <- ddp 1.1.0-rc.1 <- accounts-base 1.2.0-rc.0 <- accounts-password 1.1.0-win.1
* webapp@1.2.0-rc.0 <- routepolicy 1.0.5-rc.0 <- ddp 1.1.0-rc.1 <- accounts-password 1.1.0-win.1
* webapp@1.2.0-rc.0 <- ddp 1.1.0-rc.1 <- accounts-base 1.2.0-rc.0 <- accounts-password 1.1.0-win.1
* webapp@1.2.0-rc.0 <- ddp 1.1.0-rc.1 <- accounts-password 1.1.0-win.1
* webapp@1.2.0-rc.0 <- mongo 1.1.0-rc.0 <- accounts-base 1.2.0-rc.0 <- accounts-password 1.1.0-win.1
* webapp@1.2.0-rc.0 <- mongo 1.1.0-rc.0 <- meteor-platform 1.2.2-rc.0
* webapp@1.1.1 <- iron:router 1.0.9
* webapp@1.2.0 <- vsivsi:file-collection 1.1.1

Conflict: Constraint mongo@1.1.0 is not satisfied by mongo 1.1.0-rc.0.
Constraints on package "mongo":
* mongo@=1.1.0-rc.0 <- top level
* mongo@1.1.0-rc.0 <- meteor-platform 1.2.2-rc.0
* mongo@1.1.0-rc.0 <- autoupdate 1.2.0-win.0 <- meteor-platform 1.2.2-rc.0
* mongo@1.1.0-rc.0 <- reactive-dict 1.1.0-rc.0 <- iron:controller 1.0.8 <- iron:router 1.0.9
* mongo@1.0.8 <- dburles:mongo-collection-instances 0.3.3 <- rubaxa:sortable 1.2.0
* mongo@1.0.6 <- lai:collection-extensions 0.1.3 <- dburles:mongo-collection-instances 0.3.3 <- rubaxa:sortable 1.2.0
* mongo@1.1.0-rc.0 <- mongo-livedata 1.0.8-rc.0 <- meteorhacks:kadira 2.21.0
* mongo@1.1.0-rc.0 <- accounts-base 1.2.0-rc.0 <- accounts-password 1.1.0-win.1
* mongo@1.1.0-rc.0 <- service-configuration 1.0.4-rc.0 <- accounts-base 1.2.0-rc.0 <- accounts-password 1.1.0-win.1
* mongo@1.1.0 <- vsivsi:file-collection 1.1.1
```

And my current packages are:
```
accounts-password              1.1.0-win.1* Password support for accounts
accounts-ui                    1.1.5-rc.0* Simple templates to add login widgets to an app
autopublish                    1.0.3-rc.0* Publish the entire database to all clients
benjaminrh:jquery-cookie       0.1.0  A simple, lightweight jQuery plugin for reading, writing and deleting cookies.
coffeescript                   1.0.6-rc.0* Javascript dialect with fewer braces and semicolons
http                           1.1.0-rc.0* Make HTTP calls to remote servers
insecure                       1.0.3-rc.0* Allow all database writes by default
iron:router                    1.0.9  Routing specifically designed for Meteor
materialize:materialize        0.96.1  Materialize (official): A modern responsive front-end framework based on Material Design
meteor-platform                1.2.2-rc.0* Include a standard set of Meteor packages in your app
meteorhacks:kadira             2.21.0  Performance Monitoring for Meteor
meteorhacks:kadira-profiler    1.1.0  CPU Profiler for Kadira
meteorhacks:zones              1.4.0  Zone.Js integration for meteor
numeral:numeral                1.5.3  Numeral.js (official) - formatting and manipulating numbers.
okgrow:iron-router-autoscroll  0.0.8  Fixes page position after changing pages using Iron Router
rubaxa:sortable                1.2.0  Sortable: reactive minimalist reorderable drag-and-drop lists on modern browsers and touch devices
standard-app-packages          1.0.5-rc.0* Moved to meteor-platform
```

Any thoughts on what to do?

**Comment by vsivsi** [comment]
Hi, do you know what the significance of the `-rc.0` suffix is in the version numbers of those dependencies?

It almost seems like you are running a pre-release version of Meteor on Windows...  Or at least the packages Meteor is pulling in are prerelease... Are you running meteor from a git checkout, or via the normal install process?

Also, what version of Meteor are you running in your app? Specifically, what version is listed in `./meteor/release`?   To be current, it should be `METEOR@1.1.0.2`.

You may be able to clear this up simply by running `meteor update` in the root of your app to make sure everything is current.

**Comment by Digital-Thor** [comment]
Thanks for the quick response. I assume the `rc` suffix is for `release candidate` and something to do with the Windows version of Meteor.  As I understand it, Meteor on Windows uses different numbering so I wasn't surprised to see Meteor's version reporting as `WINDOWS-PREVIEW@0.3.0`. The Windows control panel lists the Meteor application as Version 1.0.

Nevertheless, I uninstalled Meteor, rebooted, and reinstalled using the [Windows Installer](https://www.meteor.com/install).  Then I created a new Meteor project, but still get the same errors.  According to [win-meteor](https://win.meteor.com/), I should see version 1.1, so I guess it's a problem with my set-up.  Feel free to close this.  Thanks again.
